aries-core/src/js/components/core/Identifier/Identifier.js

```diff
@@ -9,24 +10,46 @@ export const Identifier = ({
   title,
   subTitle,
   size,
+  updates,
+  type,
   ...rest
-}) => (
-  <Box align="center" {...rest}>
-    {children}
-    <Box>
-      {level ? (
-        <Heading level={level} margin="none">
-          {title}
-        </Heading>
-      ) : (
-        <Text size={size} weight="bold" color="text-strong">
-          {title}
-        </Text>
-      )}
-      <Text size={size}>{subTitle}</Text>
+}) => {
+  return (
+    <Box align="center" {...rest}>
+      <Box direction="row" justify="between" fill="horizontal">
+        {children}
+        {updates && type === 'Update' && (
```

Comment:
This seems to be a bit more particular to the updates. The intent of the Identifier component is to be a bit more generic serving as a resource identifier.

I'm open to modifying Identifier to have a region available for this type of content. Alternatively, can we get away with passing this through the `children`?